src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,11 +1,6 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
This currently violates the java coding standards: **Imported classes should always be listed explicitly.**

You can prevent IntelliJ from doing so by going through,
`Settings/Preferences -> Code Style -> Java -> Check "Use single class import"`
and
`Set Class count to use import with '*' & Names count to use static import with '*' to > 100`